psh/test-touch-rootfs.py

```diff
@@ -15,18 +15,18 @@
 from psh.tools.common import assert_mtime
 
 
-def assert_hardlinks(p):
+def assert_bin(p):
+    ''' Asserts that all executables/hardlinks available in bin are able to touch'''
```

Comment:
```suggestion
    ''' Asserts that all files in /bin are able to be touched'''
```
There might be also symlinks in /bin (I don't know if tourching symlink should change it's mtime (or the mtime of the target) so this test might still fail in the future.

Edit, it will probably fail with symlinks, there is a special option for POSIX touch not to follow symlinks:
![image](https://user-images.githubusercontent.com/3939983/221533232-68e38e27-6cfb-4606-8f48-fc1f383d30d0.png)


IMHO we can leave as is for now and worry in the future.